src/tracing/json/json_tracing.cpp

```diff
@@ -63,12 +63,25 @@ ::Json::Value GetPayload(PayloadDecoderType & decoder)
     PayloadEntry entry;
     StringBuilder<128> formatter;
 
+    int unique_idx = 0;
+
     while (decoder.Next(entry))
     {
         switch (entry.GetType())
         {
         case PayloadEntry::IMPayloadType::kNestingEnter:
             formatter.Reset().Add(entry.GetName()); // name gets destroyed by decoding
```

Comment:
This "name gets destroyed by decoding" comment does not mesh well for me with the later GetName() calls.  What is this comment saying, exactly?

Reply:
Updated code a bit here.

The issue I have is that the current name is from the "decode tree" and as we traverse the decode tree inGetPayload, the current name may change. Not sure how to express that.

Reply:
Updated comment. Hoping this is clearer. Issue is that any returned entries share a common "name buffer" to format values, so if the value requires formatting (not always as names are const c_str, but may happen for dynamic names) a call of GetPayload clobbers the previous name value.